GitExtensions/Project.Publish.targets

```diff
@@ -201,7 +217,9 @@
   <!-- Any errors in targets that executed as 'AfterTargets' don't break the build: https://github.com/microsoft/msbuild/issues/3345
        A fix is going out in VS16.6p3, but it is way too long for us to wait.
```

Comment:
With Vs17, is this still valid?

Reply:
Yes, thank you for checking. Presumably it is fixed.
The message was added in https://github.com/gitextensions/gitextensions/commit/320ecd5c7d7ba0427fb8bd0fb3ff4fc4cc77fdcd and there was a switch from `AfterTargets` to `BeforeTargets`. Switching back should work now, but it'll require testing, which I can't afford right now.